src/Billing/BillingProcessor/BillingClaimBatch.php

```diff
@@ -228,7 +234,7 @@ public function append_claim(&$segs)
 
             if ($elems[0] == 'BHT') {
                 // needle is set in OpenEMR\Billing\X125010837P
-                $this->bat_content .= substr_replace($seg, '*' . $this->bat_icn . $bat_st_02 . '*', strpos($seg, '*0123*'), 6);
+                $this->bat_content .= substr_replace($seg, '*' . $this->bat_gs06 + 1 . '*', strpos($seg, '*0123*'), 6);
```

Comment:
will this keep the padding that you need? (ie. does it stay there after it is converted to an into to be able to add 1 to it)

Reply:
oh wait, this is a replace, so forget my comment above. just move along :)